xml/System.Threading/Interlocked.xml

```diff
@@ -53,7 +53,8 @@
  The <xref:System.Threading.Interlocked.Add%2A> method atomically adds an integer value to an integer variable and returns the new value of the variable.  
   
  The <xref:System.Threading.Interlocked.Exchange%2A> method atomically exchanges the values of the specified variables. The <xref:System.Threading.Interlocked.CompareExchange%2A> method combines two operations: comparing two values and storing a third value in one of the variables, based on the outcome of the comparison. The compare and exchange operations are performed as an atomic operation.  
-  
+
+ Make sure that any write or read access to a shared variable is atomic. Otherwise, the data might be corrupted or the loaded value might be incorrect.    
```

Comment:
While this is technically correct I'm not sure if the phrasing is right for these docs, I don't think I've seen many instances of "Make sure that..." in MSDN docs.

Reply:
Probably ensure that is more common but we also have make sure that (https://github.com/dotnet/dotnet-api-docs/search?q=%22make+sure+that%22&unscoped_q=%22make+sure+that%22)

Reply:
@mairaw thanks; I go for "ensure"